Source/MonoGame.Extended/Sprites/Sprite.cs

```diff
@@ -37,7 +37,11 @@ public Sprite(Texture2D texture)
         public Vector2 OriginNormalized
         {
             get { return new Vector2(Origin.X/TextureRegion.Width, Origin.Y/TextureRegion.Height); }
-            set { Origin = new Vector2(value.X*TextureRegion.Width, value.Y*TextureRegion.Height); }
+            set {
+                  Origin = Effect == SpriteEffects.None
+                      ? new Vector2(value.X * TextureRegion.Width, value.Y * TextureRegion.Height)
+                      : new Vector2((1 - value.X) * TextureRegion.Width, value.Y * TextureRegion.Height);
```

Comment:
@craftworkgames The intention was to flip origin on flipping an image (horizontally) )) As you probably saw on my gif, it is working. I saw this solution following official Texture Packer Importer project, don't see a better option)) Also, I probably should implement it this way:
set{
Origin.X=Flip?NewOrigin.X:1-NewOrigin.X
Origin.Y=Flip?NewOrigin.Y:1-NewOrigin.Y
}
It may cause some confusion with setters & getters of Origin & OriginNormalized)))